appservice/src/deploy/runWithZipStream.ts

```diff
@@ -80,17 +69,23 @@ const commonGlobSettings: {} = {
 /**
  * Adds files using glob filtering
  */
-function addFilesGlob(zipper: archiver.Archiver, folderPath: string): void {
+export async function getFilesFromGlob(folderPath: string): Promise<string[]> {
     const zipDeployConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(ext.prefix, vscode.Uri.file(folderPath));
     const globPattern: string = zipDeployConfig.get<string>('zipGlobPattern') || '**/*';
-    const ignorePattern: string | string[] | undefined = zipDeployConfig.get<string | string[]>('zipIgnorePattern');
-    zipper.glob(globPattern, { cwd: folderPath, ignore: ignorePattern, ...commonGlobSettings });
+    const filesToInclude: string[] = await globby(globPattern, { cwd: folderPath });
```

Comment:
Did you break symlink support for App Service? I tested for Functions and it was still working, but that code is still referencing `commonGlobSettings` and this is not

Reply:
Good catch.  By default, sym links are on, but including dots are not so I enabled it in the options.

![image](https://user-images.githubusercontent.com/5290572/109859861-4db69900-7c12-11eb-8cdc-e1c1594d302c.png)

Reply:
It looks like you added it to the `filesToIgnore` but not the `filesToInclude`